dictionary/README.md

```diff
@@ -1,5 +1,9 @@
 
-Please write a couple of sentences explaining your choice of restart
-strategy.
+Choice of restart strategy for Dictionary - 
 
+Dictionary module has Supervisor has "Dictionary.Supervisor" as an children and it has configured supervision strategies as "one-for-one" i.e if a child "Dictionary" process terminates, only that process is restarted.
+	
+the supervisor restarts a child process depending on its :restart configuration.
+
+I have decided to set :restart configuration to :transient, i.e supervisor does not restart the child "Dictionary" in case it exits with reason :normal, :shutdown or {:shutdown, term}.
```

Comment:
# -2

I see your point, but in practice the Dictionary should never exit normally. In that case, :permanent and :transient are effectively the same, and :permanent better expresses your intent,